versions.json

```diff
@@ -11,8 +11,8 @@
         },
         "polymer": {
             "npmName": "@polymer/polymer",
-            "npmVersion": "3.0.5",
-            "jsVersion": "2.6.1"
+            "npmVersion": "3.1.0",
+            "jsVersion": "2.7.0"
```

Comment:
IIRC only maintenance bumps in dependencies are allowed for platform patches

Reply:
I can revert Polymer 2.6.1 but is doesn't make sense to pin 3.x at all
currently 2 versions of Polymer are being installed because of that.

Reply:
Unless things are completely broken, we should not do minor or major version bumps in platform maintenance versions.